client/stylesheets/components/_generic.scss

```diff
@@ -14,7 +14,7 @@
 
 .full.screen {
   width: 100vw;
-  height: 80vh;
+  height: 100%;
```

Comment:
1st we should change this to `min-height`

2nd this change shouldn't be there as it won't span the 100% of it's parent with this value breaking every page that uses the height of `88vh`

![example1](https://cloud.githubusercontent.com/assets/10640699/14019927/97021946-f1dd-11e5-93e1-c0160e03dd4c.png)
![example2](https://cloud.githubusercontent.com/assets/10640699/14019928/970a3a0e-f1dd-11e5-817e-bd5184a23317.png)